src/App.js

```diff
@@ -216,6 +216,7 @@ export default function App(props) {
   const [isDragging, setIsDragging] = useState(false);
   const [configToShare, setConfigToShare] = useState({ canonical: '', version: '', fhirVersion: [], dependencies: '' });
   const [sharedConfig, setSharedConfig] = useState({});
+  const [textWrapped, setTextWrapped] = useState(false);
```

Comment:
I noticed that in some places you call this text wrap and in other you call it line wrap. Unless there is a reason for the distinction, I think it might be good to call it the same thing everywhere (and I _think_ line wrap is more common).

Also, since it is a boolean, it might be more consistent with the conventions seen in the rest of this code to call it `isLineWrapped` and `setIsLineWrapped`.